pegasus/sites.v3/code.org/public/educate/it.md

```diff
@@ -42,7 +41,13 @@ For the very best experience with all Code.org content, we recommend consulting
 | **To use Web Lab** | `https://downloads.computinginthecore.org`<br/>`https://codeprojects.org`                                                                              |
 
 ## Mobile Support Details
-Our Hour of Code activities and CS Fundamentals courses can be used on mobile devices. While these activities should be usable, they are not optimized for a mobile experience and have some known limitations (e.g. touch targets may be small, accessory elements may display offscreen, and text may be difficult to input using the on-screen keyboard). Note: the CS Discoveries, CS Principles, and Intro to App Lab courses are not usable on mobile devices.
+All CS Discoveries, CS Principles, and Intro to App Lab courses are not compatible with tablets or mobile devices - a full keyboard and mouse (or touchpad, like on Macbooks) are required. Surface Pros with a keyboard attachment and Chromebooks are compatible. iPads or other tablets with a keyboard attachment are not compatible.
+
+All other content, such as K-5 courses (Computer Science Fundamentals) and Hour of Code activities are compatible on iPads and Chromebooks in addition to all other modern laptop or desktop computers.
```

Comment:
Do we need to specify that CSF is K-5? In the previous paragraph, it just says CSD, CSP, etc. Also do we need to specify iPads or can we say 'tablets' here?

Reply:
Does it seem confusing to have it listed as both K-5 and CSF? If yes, I might try to find a way to re-word it rather than remove K-5 or CSF. Reasoning: I think CSF does tend to be a little more abstract than CSD/CSP. i.e IIRC CSF includes courses 1-5, A-F, pre-reader accelerated, and accelerated. Whereas CSD and CSP always are specified as CSD/CSP.  

Re Tablets: I'm guessing iPads are intentionally called out (see reasoning above). That being said, since we exclude tablets above, we should probably include them here. What do you think of this:
"...are compatible on iPads and Chromebooks in addition to all other modern tablets and laptop or desktop computers."

Reply:
Ahh, I see what you mean on the K-5/CSF courses. My original thought was just that it stood out as the first time we referenced grades rather than courses. But based on your explanation, I think it makes sense to keep both.

Re: Tablets - I like the rephrasing you did at the bottom. It reads really smoothly!